src/components/main/ItemScreens/ItemScreen.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { useOutletContext } from 'react-router';
+
+import { ItemType } from '@graasp/sdk';
+
+import FileUploader from '@/components/file/FileUploader';
+import ItemContent from '@/components/item/ItemContent';
+import ItemMain from '@/components/item/ItemMain';
+
+import { OutletType } from './type';
+
+const ItemPage = (): JSX.Element => {
+  const { item, permission } = useOutletContext<OutletType>();
```

Comment:
Maybe `ItemContent` can use `useOutletContext` itself, so you don't need to pass all arguments from `ItemScreen`.